.eslintrc.json

```diff
@@ -27,7 +27,11 @@
           {
             "target": "./src/shadowbox/infrastructure",
             "from": ".",
-            "except": ["./src/shadowbox/infrastructure", "./node_modules"]
+            "except": [
+              "./src/shadowbox/infrastructure",
+              "./node_modules",
+              "./src/shadowbox/node_modules"
```

Comment:
Is this needed? Are the packages not created in the project root?

Reply:
Without this line, ts:lint will report invalid reference in [`follow-redirects.ts`](https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/infrastructure/follow_redirects.ts#L15) because it is referencing its local [`node-fetch` (not the root one)](https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/package.json#L15).

Reply:
removed the local node-fetch package from package.json and use the global one instead. no need to change this.